scripts/deploymentextensions/rundjangomigrations/README.md

```diff
@@ -0,0 +1,28 @@
+# RUN DJANGO MIGRATION
+
+**Function**
+This deployment extension runs django migrations for a specific app in INSTALLED_APPS of edx-platform
```

Comment:
can you provide a specific example in this comment for when we'd need to do this? it'll help future devs understand how they can use this deployment extension